app/src/services/user.ts

```diff
@@ -151,11 +151,11 @@ const service = {
    * @param {string} [defaultValue=undefined] An optional default return value
    * @returns {string} The current userId if applicable, or `defaultValue`
    */
-  getCurrentUserId: async (identityId: string, defaultValue: string | undefined = undefined) => {
+  getCurrentUserId: async (sub: string | undefined, defaultValue: string | undefined = undefined) => {
```

Comment:
Type: There should never be a situation where a subject is undefined in a JWT.
JSDoc: Don't forget to update the JSDoc param names too.